src/collections/members/ednah-akoth/index.mdx

```diff
@@ -0,0 +1,11 @@
+---
+name: Ednah Akoth
+position: Contributor, Community Member
+image_path: ./ednah-akoth.jpg
+github: Ed-Neema
+twitter: edmucious
+linkedin: ednah-akoth-7515a0207
+location: Kigali, Rwanda
```

Comment:
@Ed-Neema would you please provide a bio, so @VaibhavArora19 could add it.

Reply:
I think she is not active on github lately

Reply:
Hello @Chadha93 Apologies for the delay in response. Here's my bio write-up

I am an an undergrad Computer Science with inclinations towards Frontend development (Vue, tailwindcss, Laravel php) and Data Science. I enjoy learning new technologies and applying them to solve problems and build robust products. I have grown to relish open source contribution, not only as a means of networking with brilliant professionals and building/improving products with a global reach, but also as a means of spurring my own technical growth. Besides coding, you’d typically find me jamming on my guitar or eating.

Reply:
Thanks @Ed-Neema